data/src/main/java/com/github/leuludyha/data/repository/AuthRepositoryImpl.kt

```diff
@@ -34,6 +34,7 @@ class AuthRepositoryImpl(
             Result.Success(signInResult)
         } catch (e: Exception) {
             try {
+
                 Log.i("Auth", "Failed sign in, begin sign up")
```

Comment:
Nitpick: Try formatting your file when you finish working on it, and avoid these kind of changes to single files. Noting else changed in this file.